site/src/components/UsersTable/UsersTable.tsx

```diff
@@ -63,8 +65,9 @@ export const UsersTable: FC<UsersTableProps> = ({
         </TableRow>
       </TableHead>
       <TableBody>
-        {isLoading && <TableLoader />}
+        {isLoading && !error && <TableLoader />}
```

Comment:
On[ line 45](https://github.com/coder/coder/blob/main/site/src/pages/UsersPage/UsersPage.tsx#L45) in `UsersPage.tsx` we have:
` const isLoading = !permissions || !users || (canEditUsers && !roles)`
(which I'm not sure we want).

Anyway, I notice you're checking for the existence of `users` down below on line 142 - wanted to point out we are folding that into the loading state currently. I wonder if there's a way to clean this up a bit. What do you think?

Reply:
Thanks for pointing out, I didn't really think about it. I believe we are checking for existence since `users` can possibly be `undefined` in this file, and calling the `length` or `map` methods on it without that might disturb linting. 🤔